Assets/MixedRealityToolkit/Definitions/InputSystem/MixedRealityInputCapabilities.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Platform input capabilities.
+    /// </summary>
+    public enum MixedRealityInputCapabilities
+    {
+        ArticulatedHand = 0,
+        GGVHand,
+        MotionController,
+        EyeTracking
```

Comment:
It would be nice to add speech capabilities, as well as spatial awareness.

Reply:
FWIW I'm also fine with going with this change first to establish the framework, and then doing the speech in its own PR (smaller PRs are easier to review, from a selfish perspective)

Reply:
Looking at the other naming, since we should support those other things, we probably need to rename the capability enum to be a little more generic (i.e. not just input). Or I guess we could have two enums but that would be confusing.

Reply:
Yeah this should be a general capabilities Api, mot just related to input.

Reply:
There may well be some unestablished features that have yet to be released that may not work on hl1. It's good to be prepared

Reply:
I'll add enums for the following (and implement the checks in a future PR)

- VoiceCommand (aka keywords)
- VoiceDictation
- SpatialAwareness (question: should there be three of these? mesh, plane, point)